### ISSUE TITLE
Installing pyPLNmodels in spyder environment

### ISSUE THREAD

**Issue by mr2raccoon** [PLN-team/pyPLNmodels#4]
Hello,

 So far I was always able to install packages I needed with
```
conda activate spyder-env
conda install -n spyder-env *package* 
```
 
I tried Bastien's suggestion with:

```
conda activate spyder-env
pip install pyPLNmodels
```

It returns that requirement*s are already satisfied. Apparently, a previous try where I installed directly from the spyder console worked.  And indeed when calling `conda list`, pyPLNmodels appears under chanel `pypi `but not as the rest under `conda forge`. When calling `import pyPLNmodels` in spyder it still returns.` ModuleNotFoundError: No module named 'pyPLNmodels`. 
 
So if I understand correctly the Channel is not accesible maybe?
 
I tried solutions found [here](https://medium.com/analytics-vidhya/python-on-spyder-modulenotfounderror-a-trick-51c058129e17).
 
Unfortunately, when trying this solution it returns. `This directory cannot be added to the path! If you want to set a different Python interpreter, please go to Preferences > Main interpreter`.

Please not I consider myself still an absolute beginner in python and using anaconda / spyder. 

Any help is very much appreciated.

Best regards

Jakob

**Comment by Bastien-mva** [comment]
Hello, 

Indeed, I have tried to reproduce the error and it seems that pyPLNmodels is not installed properly on conda environments, as python does not find it. A quickfix is to run ```pip install -e .``` inside the previously cloned repository. This should be working on conda environments. 

Working on an integration with conda environments.

**Comment by Bastien-mva** [comment]
Closed with f0266fc
